docs/key-issues/cross_platform.md

```diff
@@ -9,95 +9,139 @@ most desktop platforms; however, for some platforms, this isn't the case.
 On mobile platforms, an app is compiled on a desktop platform, and transferred
 to the mobile device (or a simulator) for testing. The compiler is not executed
 on device. Therefore, it must be possible to build a binary artefact for a CPU
-architecture and a ABI that is different the platform that is running the
-compiler.
+architecture and an ABI that is different from the platform that is running the
+compiler. The situation is similar for embedded devices.
 
 Cross compilation issues also emerge when dealing with continuous
 integration/deployment (CI/CD). CI/CD platforms (such as Github Actions)
 generally provide the "common" architectures - often only x86-64 - however, a
 project may want to produce binaries for other platforms (e.g., ARM support for
-Raspberry Pi devices; PowerPC or s390 for mainframe/server devices; or for
+Raspberry Pi devices; PowerPC or s390x for mainframe/server devices; or for
 mobile platforms). These binaries won't run natively on the host CI/CD system
-(without some sort of emulation); but code can be compiled for the target
-platform.
+(without some sort of emulation, for example with QEMU); but code can be
+compiled for the target platform.
 
 macOS also experiences this as a result of the Apple Silicon transition. Apple
-has provided the tools to compile [fat binaries](multiple_architectures.md) on
-x86_64 hardware; however, in this case, the host platform (macOS on x86_64) will
-still be one of the outputs of the compilation process, and the resulting binary
-will run on the CI/CD system.
+has provided the tools to make cross compilation from x86-64 to arm64 as easy
+as possible, as well as to compile [fat binaries](multiple_architectures.md)
+(supporting x86-64 and arm64 at the same time) on x86-64 hardware. In the latter
```

Comment:
FWIW: You can compile fat binaries on M1 hardware as well - it's isn't x86_64-specific.

Reply:
Good catch, I'll fix that up.